stars/default/name.fab

```diff
@@ -274,8 +300,10 @@
  93717|h_Aql
  93429|i_Aql
 101847|l_Aql
+ 94982|A_Aql
```

Comment:
Where is source for this designation?

Reply:
```suggestion
 94982|A_Aql
```
according to [28_Aql](https://en.wikipedia.org/wiki/28_Aquilae) and [Table of Bayer designations](https://en.wikipedia.org/wiki/Table_of_stars_with_Bayer_designations)